apps/web/src/components/Logo/ChainLogo.tsx

```diff
@@ -81,6 +82,12 @@ export function getChainUI(chainId: ChainId, darkMode: boolean): ChainUI | undef
         bgColor: '#0052FF33',
         textColor: '#0052FF',
       }
+      case ChainId.HEMI_SEPOLIA: 
+      return {
+        Symbol: hemi,
+        bgColor: '#FF042033',
+        textColor: '#FF0420',
```

Comment:
these are copied from Optimism, but we may want to define the proper colors. (maybe pinging Dean)

Reply:
Changed it to match colors from hemi's site, but we should wait to see Dean's uniswap design style more accordingly.
https://github.com/hemilabs/interface/pull/1/commits/a5a5a47d4bcfde0df5da4a3e46ddc6b36e953935

Reply:
great, yeah but let's go with it for the time being